src/main/scala/it/pagopa/interop/purposeprocess/api/impl/PurposeApiServiceImpl.scala

```diff
@@ -153,7 +154,7 @@ final case class PurposeApiServiceImpl(
         else
           PurposeUpdateContentConverter.apiToDependency(purposeUpdateContent, RiskAnalysisValidation.validate).toFuture
       updatedPurpose <- purposeManagementService.updatePurpose(purposeUUID, depPayload)
-    } yield PurposeConverter.dependencyToApi(updatedPurpose)
+    } yield updatedPurpose.dependencyToApi(isRiskAnalysisValid = None)
```

Comment:
why is this `None`? shouldn't this be recalculated (`purposeUpdateContent` may contain a risk analysis form)?